keyboards/dz60/keymaps/kifinnsson/rules.mk

```diff
@@ -0,0 +1,58 @@
+# MCU name
+MCU = atmega32u4
```

Comment:
Like the `config.h` file this `rules.mk` in the keymaps directory only needs to contain things that are different from the keyboard's `rules.mk`.